dsnp/userPrivateGraph.ts

```diff
@@ -0,0 +1,49 @@
+// Paginated Chunk with PRIds
+export default {
+  type: "record",
+  name: "UserPrivateGraphChunk",
+  namespace: "org.dsnp",
+  fields: [
+    {
+      name: "pridList",
+      type: {
+        type: "array",
+        items: {
+          name: "prid",
+          type: "fixed",
+          size: 8,
+          doc: "Pseudonymous Relationship Identifier",
+        },
+      },
+    },
+    {
+      doc: "lib_sodium sealed box",
+      name: "encryptedCompressedPrivateGraph",
+      type: "bytes",
+    },
```

Comment:
Re discussion with @aramikm 

Do we need to publish the index or public key of the graph encryption key we are targeting. (`encryptedCompressedPrivateGraph` includes the random key we generated, not the key from  the PublicKey schema that we are using)